app/models/ticket.js

```diff
@@ -65,7 +65,7 @@ export default ModelBase.extend({
    * when tax amount is included in ticket price, otherwise return
    * 0
    */
-  includedTaxAmount: computed('event.tax.isTaxIncludedInPrice', 'event.tax.rate', function() {
+  includedTaxAmount: computed('price', 'event.tax.isTaxIncludedInPrice', 'event.tax.rate', function() {
```

Comment:
why include price here?

Reply:
bcz on changing the price of donation ticket, we need to update the tax amount